webapp/channels/src/components/admin_console/cluster_settings.tsx

```diff
@@ -189,6 +197,7 @@ export default class ClusterSettings extends AdminSettings {
                     onChange={this.overrideHandleChange}
                     setByEnv={this.isSetByEnv('ClusterSettings.ClusterName')}
                     disabled={this.props.isDisabled}
+                    type={'input'}
```

Comment:
explicitly passing of type input is not required because its by default input type

Reply:
done

Reply:
ok lets keep it as it is